Gemfile.lock

```diff
@@ -1,62 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.1.1)
+    better_errors (2.5.0)
       coderay (>= 1.0.0)
-      erubis (>= 2.6.6)
+      erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (9.0.6)
-    coderay (1.1.1)
-    commander (4.4.3)
-      highline (~> 1.7.2)
-    database_cleaner (1.6.1)
+    byebug (10.0.2)
+    coderay (1.1.2)
+    commander (4.4.7)
+      highline (~> 2.0.0)
+    database_cleaner (1.7.0)
     diff-lcs (1.3)
-    erubis (2.7.0)
-    event_sourcery (0.13.0)
-    event_sourcery-postgres (0.3.0)
-      event_sourcery (>= 0.10.0)
+    erubi (1.7.1)
+    event_sourcery (0.22.0)
+    event_sourcery-postgres (0.8.0)
+      event_sourcery (>= 0.14.0)
       pg
-      sequel (~> 4.38)
-    highline (1.7.8)
-    method_source (0.8.2)
-    mustermann (1.0.0)
+      sequel (>= 4.38)
+    highline (2.0.0)
+    method_source (0.9.2)
+    mustermann (1.0.3)
     pg (0.20.0)
-    pry (0.10.4)
+    pry (0.12.2)
       coderay (~> 1.1.0)
-      method_source (~> 0.8.1)
-      slop (~> 3.4)
-    pry-byebug (3.4.2)
-      byebug (~> 9.0)
+      method_source (~> 0.9.0)
+    pry-byebug (3.6.0)
+      byebug (~> 10.0)
       pry (~> 0.10)
-    rack (2.0.3)
-    rack-protection (2.0.0)
+    rack (2.0.6)
+    rack-protection (2.0.4)
       rack
-    rack-test (0.6.3)
-      rack (>= 1.0)
-    rake (12.0.0)
-    rspec (3.6.0)
-      rspec-core (~> 3.6.0)
-      rspec-expectations (~> 3.6.0)
-      rspec-mocks (~> 3.6.0)
-    rspec-core (3.6.0)
-      rspec-support (~> 3.6.0)
-    rspec-expectations (3.6.0)
+    rack-test (1.1.0)
+      rack (>= 1.0, < 3)
+    rake (12.3.1)
+    rspec (3.8.0)
+      rspec-core (~> 3.8.0)
+      rspec-expectations (~> 3.8.0)
+      rspec-mocks (~> 3.8.0)
+    rspec-core (3.8.0)
```

Comment:
### [rspec-core](https://github.com/rspec/rspec-core)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 3.6.0 → 3.8.0

[_~~change-log~~, [source-code](http://github.com/rspec/rspec-core)_]





<details>
<summary>Commits</summary>

A change of **134** commits. See the full changes on [the compare page](https://github.com/rspec/rspec-core/compare/v3.6.0...v3.8.0).

These are the first 10 commits:
- (551c365) [Bump version to 3.7.0.pre](https://github.com/rspec/rspec-core/commit/551c365197c8cae045967f872d3cd6ecf4c593a8)
- (108007e) [Frozen string literal fixes. (#2425)](https://github.com/rspec/rspec-core/commit/108007e284340d196f346ba37364801f70bfdc8a)
- (8a6544a) [Changelog for #2425](https://github.com/rspec/rspec-core/commit/8a6544a734ec5cdb769089feb318f19fbf6100a2)
- (a1c5bfd) [Reworking string handling to allow frozen string literals.](https://github.com/rspec/rspec-core/commit/a1c5bfde0e422cb040900fde84df1edba48b2c67)
- (9b774e2) [Merge pull request #2427 from pat/string-literals-2](https://github.com/rspec/rspec-core/commit/9b774e24387ca99b46a8d61a894ad89ed600895b)
- (edc7ce2) [If @pat isn't careful he may become come of us.](https://github.com/rspec/rspec-core/commit/edc7ce29f10d9d7af54ca743d76125ce130e11c4)
- (5aecd52) [Create base_formatter.rb](https://github.com/rspec/rspec-core/commit/5aecd5210958c54911f13dae6f5cc199a9d00f60)
- (bbdc984) [Add -n short option for --next-failure](https://github.com/rspec/rspec-core/commit/bbdc9849ed4f3fcad78a5afaee3e903f9fcad83d)
- (2d6180c) [Merge pull request #2434 from ianks/master](https://github.com/rspec/rspec-core/commit/2d6180c40ea12113120be0ba55b7a822ea2e0e8b)
- (ca8bc7b) [Merge pull request #2433 from lvrbrtsn/patch-1](https://github.com/rspec/rspec-core/commit/ca8bc7ba6dadb3d9b724cceca1ef67d3514ea9c5)


</details>